package.json

```diff
@@ -41,7 +41,8 @@
     "deep-equal": "^1.0.1",
     "es6-error": "^2.1.0",
     "hoist-non-react-statics": "^1.0.5",
-    "is-promise": "^2.1.0"
+    "is-promise": "^2.1.0",
+    "lodash.isequalwith": "^4.1.4"
```

Comment:
@erikras Because `lodash.*` packages inline all internal functions and variables instead of sharing them. If you have multiple `lodash.*` packages it would result in duplication.